dotCMS/src/main/java/com/dotcms/storage/DataBaseStoragePersistenceAPIImpl.java

```diff
@@ -916,4 +916,9 @@ static class PostgresUpsertDelegate extends UpsertDelegate {
 
     }
 
+    @Override
+    public String toString() {
+        return "DataBaseStoragePersistenceAPIImpl{}";
+    }
+
```

Comment:
consider giving this to String more purpose 
including some of the instance fields

Reply:
@Override
    public String toString() {
        return "DataBaseStoragePersistenceAPIImpl{" +
                "fileRepositoryManager=" + fileRepositoryManager +
                '}';
    }